test/libsolidity/semanticTests/functionCall/failed_create.sol

```diff
@@ -18,15 +18,15 @@ contract C {
 // compileViaYul: also
 // ----
 // constructor(), 20 wei
-// gas irOptimized: 265125
+// gas irOptimized: 255579
```

Comment:
Wonder what optimisation allowed this to go down by 10k.

Reply:
Previously, all the revert functions were forcefully inlined because this is how the code was generated.